### ISSUE TITLE
Non helpful error message if the SDK is not installed in the Dart preferences dialog.

### ISSUE THREAD

**Issue by gmosx** [dart-lang/eclipse3#77]
Here are the errors I got when I skipped installing the Dart SDK from within Eclipse. Not very informative:

I used the plain Eclipse Mars release.

Installed Dart Editor plugin (v1.10.1.release) from stable channel

Try to create a new project

I don't see any stagehand sample projects in the dialog

On the toolbar it says Analysis Server Inactive

When I add a dart file to the project I get:
An internal error occurred during: "Send visible files subscriptions".
java.lang.NullPointerException

When I save, I get:
An internal error occurred during: "Send visible files subscriptions".
java.lang.NullPointerException

**Comment by keertip** [comment]
@gmosx, thanks for the report. We should have a more helpful message if the SDK is not present.

Did it work once you had the SDK installed?

**Comment by gmosx** [comment]
Yeah, it worked. Very glad to see 'Dart Editor' still alive. I *really*
hope  you will keep updating this plugin, at least until dart-atom is
mature enough.
Always thought Dart Editor is underestimated.

On Wed, Sep 23, 2015 at 12:54 AM, Keerti Parthasarathy <
notifications@github.com> wrote:

> @gmosx <https://github.com/gmosx>, thanks for the report. We should have
> a more helpful message if the SDK is not present.
>
> Did it work once you had the SDK installed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dart-lang/eclipse3/issues/77#issuecomment-142433503>.
>



-- 
http://www.gmosx.com
